std/range/interfaces.d

```diff
@@ -76,8 +76,9 @@ import std.traits;
  * around input ranges with element type E.  This is useful where a well-defined
  * binary interface is required, such as when a DLL function or virtual function
  * needs to accept a generic range as a parameter. Note that
- * $(LINK2 std_range_primitives.html#isInputRange, isInputRange) and friends check for conformance to structural
- * interfaces, not for implementation of these $(D interface) types.
+ * $(REF_ALTTEXT #isInputRange, std, range, primitives, isInputRange)
```

Comment:
Wrong order here, and a bad hash sign. Should be `$(REF_ALTTEXT isInputRange, isInputRange, std, range, primitives)`.